homeassistant/components/media_player/mpd.py

```diff
@@ -89,37 +68,59 @@ def __init__(self, server, port, password, name):
         self.port = port
         self._name = name
         self.password = password
-        self.status = None
-        self.currentsong = None
-        self.playlists = []
-        self.currentplaylist = None
 
-        self.client = mpd.MPDClient()
-        self.client.timeout = 10
-        self.client.idletimeout = None
-        self.update()
+        self._status = None
+        self._currentsong = None
+        self._playlists = []
+        self._currentplaylist = None
+        self._is_connected = False
```

Comment:
Actually there is a built-in property Entity.available that IMO should be set properly based on self._is_connected, or maybe even use self.available instead of self._is_connected